lib/db/models/Pair.ts

```diff
@@ -1,26 +1,27 @@
-import Sequelize from 'sequelize';
+import { Model, Sequelize, DataTypes } from 'sequelize';
 import { getPairId } from '../../Utils';
-import * as db from '../../consts/Database';
 
-export default (sequelize: Sequelize.Sequelize, dataTypes: Sequelize.DataTypes) => {
-  const attributes: db.SequelizeAttributes<db.PairAttributes> = {
-    id: { type: dataTypes.STRING, primaryKey: true },
-    base: { type: dataTypes.STRING, allowNull: false },
-    quote: { type: dataTypes.STRING, allowNull: false },
-    rate: { type: dataTypes.FLOAT, allowNull: true },
-  };
+class Pair extends Model {
+  public id!: string;
+  public base!: string;
+  public quote!: string;
+  public rate!: number;
 
-  const options: Sequelize.DefineOptions<db.PairInstance> = {
-    tableName: 'pairs',
-    timestamps: false,
-  };
+  public static load = (sequelize: Sequelize) => {
+    Pair.init({
+      id: { type: DataTypes.STRING, primaryKey: true },
```

Comment:
I used `new DataTypes.STRING(255)` as data type for strings in the database. Could you do that in the middleware too?

If you figure out how to use `DataTypes.INTEGER` and `DataTypes.BOOLEAN` with the `new` keyword, please tell me because the sequelize documentation recommends to use `new`.

Reply:
Thanks for bringing that to my attention, `DataTypes.INTEGER` and `DataTypes.BOOLEAN` just work for me `new DataTypes.INTEGER()` and `new DataTypes.BOOLEAN()`